openssh/files/default/ssh_known_hosts

```diff
@@ -55,20 +55,20 @@
 {%- endmacro -%}
 
 {#- Pre-fetch pillar data #}
-{%- set target = salt['pillar.get']('openssh:known_hosts:target', "*.{}".format(grains['domain'])) -%}
-{%- set tgt_type = salt['pillar.get']('openssh:known_hosts:tgt_type', 'glob') -%}
-{%- set keys_function = salt['pillar.get']('openssh:known_hosts:mine_keys_function', 'public_ssh_host_keys') -%}
-{%- set hostname_function = salt['pillar.get']('openssh:known_hosts:mine_hostname_function', 'public_ssh_hostname') -%}
-{%- set use_hostnames = salt['pillar.get']('openssh:known_hosts:hostnames', False) -%}
+{%- set target = known_hosts | traverse('target', "*.{}".format(grains['domain'])) -%}
```

Comment:
what does the `known_hosts | traverse()` does? Is it supposed to be compatible with `pillar.get`?

Reply:
closing the comment as it is been discussed in 'main' conversation